plugins/node/opentelemetry-instrumentation-dns/test/functionals/utils.test.ts

```diff
@@ -154,13 +154,14 @@ describe('Utility', () => {
   describe('setError()', () => {
     it('should have error attributes', () => {
       const errorMessage = 'test error';
-      const span = new Span(
-        new BasicTracerProvider().getTracer('default'),
-        ROOT_CONTEXT,
+      const tracer = new BasicTracerProvider().getTracer('default');
+      const span = tracer.startSpan(
```

Comment:
It looks like all this test does is to check if `Span#setStatus()` and `Span#addAttributes()` is called. I wonder if we should just create mock span using `sinon` and assert that these methods are called with the expected data. This way we could eliminate the need for this test to use the SDK completely.

Reply:
@pichlermarc I think is a good idea. I'll add it in the next commit :)

Reply:
Done!